app/views/application/_nav_link.html.erb

```diff
@@ -23,7 +23,7 @@
     <%= name %>
 
     <% if pill %>
-      <span class="Nav__pill pill pill--<%= pill %> pill--small"><%= pill %></span>
+      <span class="Nav__pill pill pill--<%= pill %> pill--small" data-algolia-exclude><%= pill %></span>
```

Comment:
There's a special exclusion selector setup that means these won't be included in the DOM when Algolia does it's thing.

https://github.com/buildkite/docs/blob/main/config/algolia.json#L18